src/diffusers/models/vae.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         norm_num_groups=32,
         act_fn="silu",
         double_z=True,
+        mid_block_add_attention=True,
```

Comment:
no need to add it here no?

Reply:
I think it does need the flag because these are the encoders/decoders used by the vqvae which needs to skip the mid attention block for muse